eureka-client/src/main/java/com/netflix/discovery/shared/transport/EurekaHttpClients.java

```diff
@@ -35,7 +37,10 @@
 import com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient;
 import com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient;
 import com.netflix.discovery.shared.transport.decorator.ServerStatusEvaluators;
+import com.netflix.discovery.shared.transport.jersey.EurekaJerseyClient;
```

Comment:
EurekaHttpClients should have no dependency on any transport implementation, as it is part of transport API.